d2l-floating-buttons.html

```diff
@@ -82,6 +82,11 @@
 			is: 'd2l-floating-buttons',
 
 			properties: {
+				alwaysFloat: {
+					reflectToAttribute: true,
+					type: Boolean,
+					value: false
```

Comment:
Can the default value `false` be omitted here?  It's not a concern for this component since it's only used once per page, but during the component perf work I identified it as sometimes being a significant portion of the start-up cost for the component, so I've generally avoided the default unless it is necessary.